python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(self.gangadir, ignore_errors=True)
```

Comment:
Fixed the typo. Interesting we have lost the logging by this point so debugging that may have been difficult